CHANGELOG.md

```diff
@@ -8,12 +8,12 @@ This release adds support for custom paths to the configuration file, support fo
 
 #### :gear: Specify Custom Path to the TestCafe Configuration File ([PR #6035](https://github.com/DevExpress/testcafe/pull/6035) by [@Chris-Greaves](https://github.com/Chris-Greaves))
 
-TestCafe now allows you to specify a custom path to the [configuration file](../documentation/reference/configuration-file.md).
+TestCafe now allows you to specify a custom [configuration file](https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html) path.
 
 To set the path, use one of the following options:
 
-* the [--config-file CLI option](../documentation/reference/command-line-interface.md#--config-file-path)
-* the [configFile option of the createTestCafe function](../documentation/reference/testcafe-api/global/createtestcafe.md#options)
+* the [--config-file CLI option](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html#--config-file-path)
```

Comment:
Since you already used the word option in the sentence describing this list, don't use it in list. Rewrite this part.

Reply:
> * the [--config-file CLI flag](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html#--config-file-path)